nuttx-configs/px4fmu-v4pro/nsh/defconfig

```diff
@@ -627,7 +627,7 @@ CONFIG_BOOT_RUNFROMFLASH=y
 # Boot Memory Configuration
 #
 CONFIG_RAM_START=0x20000000
-CONFIG_RAM_SIZE=327680
```

Comment:
We need to see if there is a better way to do this, so there is only one source. I will open an issue.